tests/outlineCompiler_test.py

```diff
@@ -1075,7 +1079,12 @@ def test_custom_layer_compilation_interpolatable_from_ds(designspace, inplace):
         "edotabove",
     ]
 
-    sparse_tables = [tag for tag in master_ttfs[1].keys() if tag != "GlyphOrder"]
+    sparse_tables = [
+        tag
+        for tag in master_ttfs[1].keys()
+        if tag not in ("GlyphOrder", "cvt ", "fpgm", "prep")
+    ]
+    # raise NotImplementedError(f"{SPARSE_TTF_MASTER_TABLES} issuperset {sparse_tables}")
```

Comment:
what does this commented out line mean?

Reply:
Honestly I don’t remember. Probably some leftover of  debugging.